src/js/util/svg.js

```diff
@@ -71,10 +71,7 @@ define(function (require, exports) {
         _.forEach(filter, function (kind) {
             var iconID = "layer-";
 
-            // No svg for these
-            if (kind === "solid color" || kind === "gradient" || kind === "pattern") {
-                iconID = "tool-rectangle";
-            } else if (kind === "artboard") {
+            if (kind === "artboard") {
```

Comment:
Same here, not sure if artboard is an acceptable kind, I ran a quick string search and don't see any `artboard` string being added anywhere. Can you correct it here, or correct me? :)